src/libsyncengine/syncpal/tmpblacklistmanager.cpp

```diff
@@ -90,9 +90,9 @@ void TmpBlacklistManager::blacklistItem(const NodeId &nodeId, const SyncPath &re
         errorInfo._path = relativePath;
         errors.insert({nodeId, errorInfo});
 
-        LOGW_DEBUG(Log::instance()->getLogger(), L"Item added in " << (side == ReplicaSideLocal ? L"local" : L"remote")
-                                                                   << L" error list with nodeId=" << Utility::s2ws(nodeId).c_str()
-                                                                   << L" and path=" << Path2WStr(errorInfo._path).c_str());
+        LOGW_INFO(Log::instance()->getLogger(), L"Item added in " << (side == ReplicaSideLocal ? L"local" : L"remote")
```

Comment:
Would it be possible to re-use a lambda or a method for this logging operation which seems identical to the above one?

Reply:
Done

Reply:
Done